rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -1,32 +1,22 @@
+#
+#  Copyright (C) 2011-2017 Greg Landrum
```

Comment:
Good suggestion, but I don't think it's that important (and I don't think it's a good idea to update the copyright on files that haven't otherwise been changed). I currently update things when it feels like I've changed a fair amount. Oh, and I remember. :-S